pkg/identity/key/global_identity.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package key
+
+import (
+	"strings"
+
+	"github.com/cilium/cilium/pkg/allocator"
+	"github.com/cilium/cilium/pkg/labels"
+)
+
+// GlobalIdentity is the structure used to store an identity
+type GlobalIdentity struct {
+	labels.LabelArray
+}
+
+// GetKey encodes an Identity as string
+func (gi GlobalIdentity) GetKey() string {
```

Comment:
Any reason this isn't a pointer receiver?

Reply:
(I see this was pre-existing code, but I can't see an obvious reason for this **not** to be a pointer)

Reply:
Changed it to the pointer receiver.